.github/workflows/continous-integration.yml

```diff
@@ -139,12 +139,18 @@ jobs:
       with:
         poetry-version: ${{ env.POETRY_VERSION }}
 
+    - name: Set up virtual environment (Windows)
+      if: matrix.os == 'windows-latest'
+      run: |
+        poetry config virtualenvs.in-project true
+
     - name: Load Poetry Cached Libraries ⬇
+      if: matrix.os == 'windows-latest'
       uses: actions/cache@v1
       with:
-        path: ~/.cache/pypoetry/virtualenvs
-        key: ${{ runner.os }}-poetry-${{ matrix.python-version }}-${{ hashFiles('**/poetry.lock') }}
-        restore-keys: ${{ runner.os }}-poetry-${{ matrix.python-version }}
+        path: .venv
```

Comment:
Is `.venv` the default for poetry? In their docs they say the default virtualenv path is `{cache-dir}\virtualenvs` 🤔

Reply:
It's default when you use set `virtualenvs.in-project` to `true`. I will add a comment